client/src/components/Applications/AppForm/fixtures/AppForm.app.fixture.js

```diff
@@ -170,17 +193,45 @@ export const helloWorldAppFixture = {
         minMinutes: 1,
         maxMinutes: 120,
       },
+      {
+        name: 'small',
+        hpcQueueName: 'small',
+        maxJobs: -1,
+        maxJobsPerUser: 20,
+        minNodeCount: 1,
+        maxNodeCount: 2,
+        minCoresPerNode: 1,
+        maxCoresPerNode: 56,
+        minMemoryMB: 1,
+        maxMemoryMB: 192000,
+        minMinutes: 1,
+        maxMinutes: 2880,
+      },
+      {
+        name: 'rtx',
```

Comment:
@rstijerina , I just did a quick update of this and the small queue (so they might not be accurate).  so we should consider updating this whenever we have those queue definitions.  or do you know if those already exist somewhere?

Reply:
Cool, yeah I haven't updated the system to reflect the full queue list yet